include/unicore-mx/usbd/usbd.h

```diff
@@ -279,7 +284,7 @@ enum usbd_transfer_flags {
 	USBD_FLAG_PACKET_PER_FRAME_MASK = (0x3 << 5)
 };
 
-typedef enum usbd_transfer_flags usbd_transfer_flags;
```

Comment:
@alexrayne https://stackoverflow.com/questions/17918380/typedef-and-enum-with-same-name